lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -2313,6 +2313,60 @@ def _containers(self) -> Dict[str, Container]:
         return {cont: self._charm.unit.get_container(cont) for cont in self._logs_scheme.keys()}
 
 
+class PebbleLogForwarder:
```

Comment:
What do you think about calling it something like `StdoutForwarder`?

Reply:
I don't mind; it probably makes what it does more clear

Reply:
We just agreed to name it `LogForwarder`.